### ISSUE TITLE
Could not find 'eventmachine' (= 1.0.9.1) - did find: [eventmachine-1.2.5]

### ISSUE THREAD

**Issue by stefanhuber** [sj26/mailcatcher#349]
It seems that mailcatcher doesn't currently work with the newest release of eventmachine. I am using it on ubuntu 16.04. Somehow eventmachine got updated on my machine...

I solved the problem in this way:
```
$ gem uninstall eventmachine
$ gem install eventmachine -v 1.0.9.1
```

Full error message:
```
/home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems/dependency.rb:310:in `to_specs': Could not find 'eventmachine' (= 1.0.9.1) - did find: [eventmachine-1.2.5] (Gem::MissingSpecVersionError)
Checked in 'GEM_PATH=/home/stefan/.rvm/gems/ruby-2.3.4:/home/stefan/.rvm/gems/ruby-2.3.4@global', execute `gem env` for more information
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb:1439:in `block in activate_dependencies'
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb:1428:in `each'
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb:1428:in `activate_dependencies'
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb:1410:in `activate'
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems.rb:299:in `block in activate_bin_path'
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems.rb:299:in `synchronize'
	from /home/stefan/.rvm/rubies/ruby-2.3.4/lib/ruby/site_ruby/2.3.0/rubygems.rb:299:in `activate_bin_path'
	from /home/stefan/.rvm/gems/ruby-2.3.4/bin/mailcatcher:22:in `<main>'
	from /home/stefan/.rvm/gems/ruby-2.3.4/bin/ruby_executable_hooks:15:in `eval'
	from /home/stefan/.rvm/gems/ruby-2.3.4/bin/ruby_executable_hooks:15:in `<main>'

```

**Comment by michaelschmitz** [comment]
Same here - thanks for the solution. Seems like mailcatcher is set to fixed 1.0.9.1 instead of max version. Everytime I update my gems I now have to run this workaround. Might be time to change the dependency in mailcatcher.

**Comment by sj26** [comment]
Mailcatcher is locked to a particular version of eventmachine because it tends to break very easily. Unfortunately rubygems doesn't seem to respect intergem dependencies very well so the wrong versions are often activated. :-(

**Comment by sj26** [comment]
Your solution here is the correct solution, for now:

```
$ gem uninstall eventmachine
$ gem install eventmachine -v 1.0.9.1
```

Longer term this should be solved by #298
